tekton/ci-workspace/jobs/tekton-catalog-catlin-lint.yaml

```diff
@@ -50,7 +50,7 @@ spec:
         echo '```' >> catlin.txt
 
         # performing catlin validate
-        catlin validate changed-files.txt | tee -a catlin.txt
+        catlin validate $(cat changed-files.txt) | tee -a catlin.txt
```

Comment:
or 

```bash
catlin validate < change-files.txt | tee -a catlin.txt
```

@chmouel wdyt ?

Reply:
@vdemeester catlin doesn't takes input like this as of now and gives the following error
```
Error: requires at least 1 path to tekton resource yaml but received none
```
Probably we can update `catlin` in a follow-up PR

Reply:
ohhh I see 😓

Reply:
or we can do the same as : 

https://github.com/tektoncd/plumbing/pull/839/files#diff-b84a0238906120be8e62bed805b8a3aaf596c2741c671f40fde8306c2704d60eL57-L59

Reply:
@chmouel catlin validate command can accept an array of args so we can run `catlin validate task/git-clone/0.2 task/black/0.1` whereas `catlin lint script` accepts only one arg at a time hence the for loop was there and with `catlin validate` I passed the whole string from the file :sweat_smile: